app-dev/video-to-blog/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,96 @@
+# Code of Conduct
```

Comment:
This repo itself already has a code of conduct, license, contributing docs, etc. (https://github.com/GoogleCloudPlatform/devrel-demos/tree/main/.github)

Is the duplication intentional?